tools/docker/corda-all-in-one/corda-v5/cordapps-deployment/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM docker:20.10.3-dind
```

Comment:
@adrianbatuto why do we have this Dockerfile seperate from the one present in `corda-all-in-one/corda-v5/` folder? I think we should have this Dockerfile as the only dockerfile, as the difference between the 2 is very minimal (one just have some extra env vars and a binary moved locally)